.github/workflows/deploy-with-helios.yml

```
@@ -18,12 +18,20 @@ on:
         type: string
+      app_version:
+        description: "Optional runtime version (defaults to the provided commit SHA)"
+        required: false
+        type: string
 
 # Suggestion: Ensures only one workflow runs at a time for a given environment name
 concurrency: ${{ github.event.inputs.environment_name }}
 
 jobs:
   deploy-app:
+    env:
```

actionlint (1.7.8)

when a reusable workflow is called with "uses", "env" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "deploy-app"

(syntax-check)